tests/internal/e2elib/e2elib.go

```diff
@@ -43,7 +43,7 @@ const (
 
 // By default, kind logs are collected when the e2e tests fail. The TEST_KEEP_CLUSTER environment variable
 // can be set to "true" to preserve the logs and the kind cluster even if the tests pass.
-var keepCluster = func() bool {
+var KeepCluster = func() bool {
```

Comment:
making it accessible to test so that they can use it on individual cleanups to skip removing resources.